cScripts/cScripts_pylons.sqf

```diff
@@ -176,5 +176,260 @@
+                ["rhs_mag_100rnd_127x99_mag_Tracer_Red",[0],100],
+                ["rhs_mag_100rnd_127x99_mag_Tracer_Red",[0],100],
+                ["rhs_mag_100rnd_127x99_mag_Tracer_Red",[0],100],
+                ["rhs_mag_100rnd_127x99_mag_Tracer_Red",[0],100],
+                ["rhs_mag_100rnd_127x99_mag_Tracer_Red",[0],100],
+                ["rhs_mag_100rnd_127x99_mag_Tracer_Red",[0],100],
+                ["rhs_mag_100rnd_127x99_mag_Tracer_Red",[0],100],
+                ["rhs_mag_100rnd_127x99_mag_Tracer_Red",[0],100],
+                ["rhs_mag_100rnd_127x99_mag_Tracer_Red",[0],100],
+                ["rhs_mag_100rnd_127x99_mag_Tracer_Red",[0],100],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200]
+            ]]
+        ]]
+    ]],
+    ["rhsusf_M1165A1_GMV_SAG2_Mk19_M240_base", [
+        ["default",[
+            ["displayName", "Default"],
+            ["icon", ""],
+            ["loadout",[
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
```

Comment:
Magazines are 48 rounds any reason why we do 100?

Reply:
That's a valid point... I'm gonna change that